tests/test_resonance_scattering/test_resonance_scattering.py

```diff
@@ -1,60 +1,10 @@
 #!/usr/bin/env python
 
-import os
 import sys
-from subprocess import Popen, STDOUT, PIPE, call
-import filecmp
-import glob
-from optparse import OptionParser
+sys.path.insert(0, '..')
```

Comment:
The insertion adds `testing_harness` to the path.  `import testing_harness` will fail without that line.